src/html/index.html

```diff
@@ -0,0 +1,71 @@
+{#
+  Nunjucks Templating!
+  Homepage: https://mozilla.github.io/nunjucks/
+  Docs: https://mozilla.github.io/nunjucks/api.html
+#}
+
+{% extends 'layouts/application.html' %}
+
+{% block content %}
+<div id="top-fold">
+  <div id="title-box">
+  <p id="subtitle">A Viget Exploration</p>
+    <h1 id="title">
+      <a href="/">
+        <img src="images/title.svg" alt="Improving Medical Adherence" />
+      </a>
+    </h1>
+    <img src="images/yellow-pill.svg" alt="Yellow pill." id="yellow-pill">
+    <img src="images/pills.svg" alt="Green and light yellow pills." id="pills">
+    <img src="images/more-pills.svg" alt="Three pill tablets." id="more-pills">
+    <img src="images/purple-pills.svg" alt="Two purple pills." id="purple-pills">
```

Comment:
might be interesting to try doing these pills with pseudo elements since they're purely decorative

Reply:
Eventually, Peyton will be adding some animation and movement. I'd hold off on changing this until an appropriate effect and technical approach is decided.